portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,17 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random info to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomInfo() {
+  const infos =
+      ["I'm student at Dalian University of Technology.", 
+       "The pronunciation of my Chinese name is similar to '复习英语'(Review English)."];
```

Comment:
It would be more interesting if you could add more info sentences.

Reply:
I agree. I'll add more info sentences.